### ISSUE TITLE
Color contrast ratio

### ISSUE THREAD

**Issue by cpeartree** [cpeartree/Christopher_Portfolio_2#13]
**NAME:** 
Color contrast ratio

**STATUS:** 
Assigned (Christopher Poirier)

**DESCRIPTION:** 
Button background and foreground colorus lack sufficient contrast ratio
site-header background and li items lack sufficient contrast ratio

![Image](https://user-images.githubusercontent.com/107279057/227379905-bbbfc4b4-064b-49dd-bb45-88145e8e8e64.png)

![Image](https://user-images.githubusercontent.com/107279057/227379912-e1fd9cf1-6b5e-4f31-a609-51a38a00a796.png)

![Image](https://user-images.githubusercontent.com/107279057/227380563-aa3a4de2-a966-4fca-91a3-1400af63b374.png)

**STEPS TO REPRODUCE:** 
- Run page through [Google Page Speed Insights ](https://pagespeed.web.dev/) 

**EXPECTED RESULTS:**
- Content color contrast ratio follows accessibility best practices to allow for readability. Reference: [A11y-101](https://a11y-101.com/design/contrast)

**CONFIGURATION:**
- This issue exists in the source code, and will be viewable on all browser/system configurations

**SEVERITY:**
- High

**PRIORITY:**
- High

**TYPE:** 
- Bug

**ASSIGNEE:**
- Christopher Poirier
